### PR TITLE
Deploy Nephio workload

### PR DESCRIPTION
**Pull request by electrocucaracha**
This change the set of instructions to test the workload provisioning process through Cluster API, these steps will be moved to integration tests later.

### CODE REVIEW

File: e2e/provision/gce_run.sh [hunk 1]
```diff
@@ -44,20 +45,31 @@ if ! lsmod | grep -q gtp5g; then
...
-#else
-#    ansible-playbook -vvv -i ~/nephio.yaml playbooks/cluster.yml
-#fi
+if [ "${DEPLOYMENT_TYPE:-one-summit}" == "one-summit" ]; then
+    [[ -d "$HOME/workshop" ]] || git clone --depth 1 https://github.com/nephio-project/one-summit-22-workshop.git "$HOME/workshop"
+    mkdir -p "$HOME/workshop/nephio-ansible-install/inventory"
+    cp "$HOME/nephio.yaml" "$HOME/workshop/nephio-ansible-install/inventory/"
+    pushd "$HOME/workshop/nephio-ansible-install" >/dev/null
+    for playbook in install-prereq create-gitea create-gitea-repos deploy-clusters configure-nephio; do
+        if [[ ${DEBUG:-false} != "true" ]]; then
```
Comment (radoslawc) [inline e2e/provision/gce_run.sh:54]: Nice

**Review by radoslawc — COMMENTED** [review]
/approve

**Comment by vjayaramrh** [comment]
REviewing in progress currently from my side, any instructions (if any) that you may have to test this PR out in the PR description would be helpful.

**Pull request merged**